cmd/runtimetest/main.go

```diff
@@ -320,6 +320,14 @@ func testWriteAccess(path string) error {
 	return nil
 }
 
+func validatePlatform(spec *rspec.Spec) error {
+	if runtime.GOOS != "linux" && runtime.GOOS != "solaris" && runtime.GOOS != "windows" {
+		return fmt.Errorf("host platform %q is not supported", runtime.GOOS)
```

Comment:
This is presumably based on [this spec section][1], but there's no RFC 2119 language there, and the platform target is closer to selecting the relevant spec than it is to being a requirement of the spec.  I'm fine with having something like this being a fatal error (i.e. “runtime-tools has not implemented testing for your platform [because the spec has nothing to say about it][1]”).  But I don't see a point to making it soft error after which we continue on to run our other tests.  If we're comfortable running the other tests, we should just run them, and let each test take care of skipping all the things that are specific to one of the above platforms (which is most things).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/spec.md#platforms

Reply:
updated, PTAL.